app/components/EventItem/styles.css

```diff
@@ -6,6 +6,13 @@
   display: flex;
   justify-content: space-between;
   line-height: 1.3;
+  transition: var(--easing-fast);
+}
+
+.eventItemHover {
+  background-color: var(--additive-background);
+  border-top-right-radius: 10px;
+  border-bottom-right-radius: 10px;
```

Comment:
Do we only want border on the right side? This feels kinda off from the rest of the webapp. Besides, please use border radius variables like `var(--border-radius-sm)` for this instead of `10px`😅

Reply:
I will fix border radius with the variable instead of 10px. But about if we should have the border radius or not, if you look at the job listing page, there are border radiuses on those elements. So I am not entirely sure what is most consistent, no border radius or border radius.

Reply:
The items on the joblisting page also has border radiuses. I think we should do the same here, but I was just contemplating wether we only wanted it on the right side. The colored border on the left side is kind off atm